src/hildebrand/internal/response.cljc

```diff
@@ -143,7 +155,7 @@
   (with-meta (or items []) (dissoc m :items)))
 
 (defn error [type message & [data]]
-  (assoc data :hildebrand/error {:type type :message message}))
+  {:hildebrand/error (assoc data :type type :message message)})
```

Comment:
If there are any callers passing a value for `data`, this change could be a problem - either way, the return value will have a hildebrand/error key, so we should be OK without this?